templates/definition/meter/sunspec-hybrid.yaml

```diff
@@ -65,50 +65,45 @@ render: |
   power:
     source: calc
     add:
-    - source: modbus
+    - source: sunspec
       {{- include "modbus" . | indent 4 }}
-      model: sunspec
-      value: DCPowerS1 # mppt 1
-    - source: modbus
+      value: 160:1:DCW # mppt 1
+    - source: sunspec
       {{- include "modbus" . | indent 4 }}
-      model: sunspec
-      value: DCPowerS2 # mppt 2
+      value: 160:2:DCW # mppt 2
   energy:
     source: calc
     add:
-    - source: modbus
+    - source: sunspec
       {{- include "modbus" . | indent 4 }}
-      model: sunspec
-      value: DCEnergyS1 # mppt 1
-    - source: modbus
+      value: 160:1:DCWH # mppt 1
+      scale: 0.001
```

Comment:
Weshalb diese Skalierung?

Reply:
Weil wir Energie in kWh messen

Reply:
Ah, ok, vorher war das schon von mbmd vorskaliert, oder?

Reply:
Genau. Alles schwarze Magie...